RedHat-UX/red-hat-design-system
# docs(badge): update best practices

## What I did

1. Used Do/Don't format for badge's "Best practices" section


## Testing Instructions

1. Check badge's ["Best practices" section](https://deploy-preview-2099--red-hat-design-system.netlify.app/elements/badge/guidelines/#best-practices) in DP

## Notes to Reviewers
